templates/commitlint.config.js

```diff
@@ -12,7 +12,7 @@ const Configuration = {
     'subject-full-stop': [2, 'never', '.'],
     'type-case': [2, 'always', 'lower-case'],
     'type-empty': [2, 'never'],
-    'type-enum': [
+    'type-enum': [2, 'always', [
```

Comment:
@d3viant0ne is this correct? my review was requested but I dunno much about this

Reply:
It is, commitlint API changed. That was that error you got in the CLI regarding enum